content/en/integrations/guide/pivotal-cloud-foundry-manual-setup.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Pivotal Cloud Foundry Manual Setup Guide
+title: Cloud Foundry Manual Setup Guide
```

Comment:
```suggestion
title: Cloud Foundry Setup Guide
```
We don't need to specify `manual` since it's the only way for the open source cloud foundry.